### PR TITLE
Web Component Changes

### PR DESCRIPTION
**Pull request by beaversd**
Changes made to the web components to properly style the cytoscape map.

### CODE REVIEW

File: cytoscapeResize.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from asyncore import read
```
Comment (guanmingwu) [inline cytoscapeResize.py]: Do you need this package?

File: src/components/features/PairwiseSearch/Cytoscape/PathwayGeneSimilarity.vue [hunk 2]
```diff
@@ -1,66 +1,78 @@
...
+        style="top: 4px; right: 4px; position: absolute; align-items: center;">
+      </LegendTable>
     </v-expand-transition>
   </v-container>
 </template>
 
 <script>
 
-import { mdiCogOutline, mdiMapLegend } from '@mdi/js';
+import { mdiCogOutline, mdiMapLegend, mdiMenuDown } from '@mdi/js';
```
Comment (guanmingwu) [inline src/components/features/PairwiseSearch/Cytoscape/PathwayGeneSimilarity.vue]: mdiMenuDown is not used any more. Right?

File: src/components/features/PairwiseSearch/Cytoscape/PathwayGeneSimilarity.vue [hunk 3]
```diff
@@ -83,12 +95,18 @@ export default {
...
+    isWebComponent:{
+      type: Boolean,
+      default: false,
     }
   },
   // This is a function
   data: () => ({
     mdiCogOutline,
     mdiMapLegend,
+    mdiMenuDown,
```
Comment (guanmingwu) [inline src/components/features/PairwiseSearch/Cytoscape/PathwayGeneSimilarity.vue]: See the previous question.

File: src/components/features/PairwiseSearch/Cytoscape/PathwayGeneSimilarity.vue [hunk 4]
```diff
@@ -284,30 +304,34 @@ export default {
 };
 </script>
 
-<style style scope>
+<!-- <style style scope>
```
Comment (guanmingwu) [inline src/components/features/PairwiseSearch/Cytoscape/PathwayGeneSimilarity.vue]: Let's just delete this section if nothing is used here?

File: cytoscapeResize.py (hunk 1 above)
Comment (beaversd) [reply cytoscapeResize.py]: No, I will delete that.

**Pull request merged**